app/src/views/OperationalLearning/index.tsx

```diff
@@ -270,19 +245,40 @@ export function Component() {
         query,
     ]);
 
+    const handleApplyFilters = useCallback(() => {
+        triggerSummary(query);
+        triggerOperationalLearning(query);
```

Comment:
if we switch to useRequest, it will trigger automatically once the dependent filters are changed